playground/models/networks/sampling.py

```diff
@@ -23,6 +39,11 @@ def sample_standard_normal_noise(inputs, **kwargs):
         resized_noise = Dense(data_dim, activation=None, name='enc_resized_noise_sampler')(samples_isotropic)
         added_noise_data = Add(name='enc_adding_noise_data')([inputs, resized_noise])
         return added_noise_data
+    elif op_mode == 'product':
+        outputshape = noise_dim**2
+        bilinearProduct = Lambda(outer_product, output_shape=(noise_dim ** 2, ))([inputs, samples_isotropic])
```

Comment:
change the variable name to PEP style convention using underscores to `bilinear_product`. Also why is it called bilinear product?